### DISCUSSION TITLE
[J 4.3.x] Language String for Menu Item Type Headline Articles

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#43390] [2. Cold Features]
### What needs to be fixed
An interesting finding of the Joomla Kids: When you create a new Menu Item, the headline of the menu Item types for Articles and Categories is "Articles" instead of "Content". That's not logical because Categories are below Content (Backend Menu) 
<img width="369" alt="grafik" src="https://github.com/joomla/joomla-cms/assets/828371/2c632442-b58e-494a-adfc-651e83396c4f">

### Why this should be fixed
Because it follows more the logic of the Joomla Structure

### How would you fix it
Change the Headline Articles to Content

### Side Effects expected
Unfortunately, a lot of documentation screenshots or explanations to be corrected. But honestly, I think we should change it to be more consistent.

**Comment by brianteeman** [comment]
Articles is correct.

Content is a heading just like components

Content includes media and since you changed it for j4 modules as well

**Comment by coolcat-creations** [comment]
Yes but Articles does not contain categories, that does not make sense.
I would chose the heading Content because its com_content and not com_articles

**Comment by chmst** [comment]
Agree with @coolcat-creations.  It is not obvious if you are new to Joomla that Categories are beblow Articles. Categories contain articles, not the other way around. 

Content is not quite accurate either, as in a content managment system everything is content but it would match the menu.

**Comment by brianteeman** [comment]
Sorry but you are wrong.

OR do you want to take categories out of articles/banners/contacts etc

**Comment by brianteeman** [comment]
or I have completely misunderstood what you are saying

**Comment by chmst** [comment]
Sorry for poor english. No, it was just a remark that in a content management everything is "content". 

I agree with @coolcat-creations. "Content" instead of "Articles"

**Comment by brianteeman** [comment]
sigh
If everything is content in a CMS (i agree) then even more reason that this should be specific and say articles and not content

**Comment by brianteeman** [comment]
what is there in that menu type that is not exclusively for displaying or editing articles

**Comment by coolcat-creations** [comment]
The name of the Component is com_content and not com_articles. That should be reason enough to show that it's not logical to write "Articles" there. Articles can be in Categories but Categories can't be in Articles.

**Comment by brianteeman** [comment]
You really need to think about this a bit more

**Comment by brianteeman** [comment]
![image](https://github.com/joomla/joomla-cms/assets/1296369/86c65534-2322-4556-9616-8004fcfe7cd0)

![image](https://github.com/joomla/joomla-cms/assets/1296369/2133cde4-8e05-4c58-98f8-b5362a0a50f3)

![image](https://github.com/joomla/joomla-cms/assets/1296369/354e6ee7-240d-47ca-bd22-38c583502ceb)

and the internal name of the component is irrelevant
com_finder = Smart Search
com_config = Configuration

**Comment by coolcat-creations** [comment]
I suppose we are lost in translation and I could not describe the issue properly.

If I want to add a new Menu item type for Kunena, the Headline of the selection is Kunena, just like the Components Name
If I want to add a new Menu item type for User, the Headline of the selection is User, just like the Components Name
If I want to add a new Menu item type for Tags, the Headline of the selection is Tags, just like the Components Name 

If I want to add a new Menu item type for com_content, I would search for Content as headline and not for one small part of com_content

**Comment by brianteeman** [comment]
> If I want to add a new Menu item type for com_content, I would search for Content as headline and not for one small part of com_content

1. How would you even know it is coded as com_content - the name of the component is Articles
2. small part??

**Comment by chmst** [comment]
I think th components name is not relevant, users in general don't know the component name.

It should be consistent with the name of the corrresponding main menu item, no matter what is in the code.

**Comment by coolcat-creations** [comment]
> It should be consistent with the name of the corrresponding main menu item, no matter what is in the code.

Which would be Content as well

**Comment by brianteeman** [comment]
> I think th components name is not relevant, users in general don't know the component name.

agree

> 
> It should be consistent with the name of the corrresponding main menu item, no matter what is in the code.

as I stated before that logic would mean all components would be under a new components container

**Comment by coolcat-creations** [comment]
>     1. How would you even know it is coded as com_content - the name of the component is Articles
Because of the headline in the admin menu

>     2. small part??
yes because Categories are another part

**Comment by coolcat-creations** [comment]
> as I stated before that logic would mean all components would be under a new components container

The menu item of Smart Search is Smart Search (in the second level)

**Comment by brianteeman** [comment]
> headline in the admin menu

that is NOTHING to do with the component name see smart search as an example

**Comment by coolcat-creations** [comment]
Smart Search as example:
<img width="291" alt="grafik" src="https://github.com/joomla/joomla-cms/assets/828371/553c80a3-2e29-441c-bf72-3f171c91ca6f">

**Comment by coolcat-creations** [comment]
Another Solution would be to create a new Group named "Categories" and place there all com_content Category stuff in this group.

**Comment by chmst** [comment]
![grafik](https://github.com/joomla/joomla-cms/assets/1035262/493444d6-0754-4a60-ae5c-820317efdc9b)

I think this is what users see. The name of the component in the code doesn't matter

**Comment by coolcat-creations** [comment]
Thank you @chmst 
Yes thats why I said in the initial post "That's not logical because Categories are below Content (Backend Menu)"

**Comment by brianteeman** [comment]
Thanks @chmst you prove my point.

You are expanding the Components menu 
You are Not expanding the Content menu

**Comment by coolcat-creations** [comment]
Then lets create a second menu item type group "Categories"

**Comment by brianteeman** [comment]
and are you also going to do the same for Banners, contacts, newsfeeds

makes no sense

**Comment by brianteeman** [comment]
This is the exact type of pointless UI change that users object to - especially after 17 years

**Comment by coolcat-creations** [comment]
> and are you also going to do the same for Banners, contacts, newsfeeds
> 
> makes no sense

Makes more sense to me to Name it Content. I agree :-)

**Comment by chmst** [comment]
It is an UX issue and it would be much better to ask users

**Comment by coolcat-creations** [comment]
> This is the exact type of pointless UI change that users object to - especially after 17 years

Sure, after 17 years, it would be a shame if we suddenly became interested in progress and improvements. 👎

**Comment by brianteeman** [comment]
> It is an UX issue and it would be much better to ask users

after 17 years with no one else saying anything dont you think they have already spoken?

time for me to bow out of Joomla

**Comment by coolcat-creations** [comment]
> time for me to bow out of Joomla

I'm sorry you feel that way. My goal was not to upset you or make you leave Joomla. This comment because of a headline changes is shocking me. I think our differing opinions show how passionate we both are about finding the best solutions for Joomla and its users. Let's get together to discuss the pros and cons of the proposed changes and look for a compromise that we're all happy with. In the end, we all want what's best for Joomla and a growing user base. I think that the kids had a very valid point and maybe we can do something to make new users understand Joomla easier.

**Comment by chmst** [comment]
> after 17 years with no one else saying anything dont you think they have already spoken?

Finally someone said what I have thought so often. I think we should take  these issues which come directly from users seriously and be open for improvement.
